DataFormats/HGCalReco/interface/TICLSeedingRegion.h

```diff
@@ -0,0 +1,29 @@
+// Authors: Felice Pantaleo, Marco Rovere
+// Emails: felice.pantaleo@cern.ch, marco.rovere@cern.ch
+// Date: 06/2019
+
+#ifndef RecoHGCal_HGCalReco_TICLSeedingRegion_h
+#define RecoHGCal_HGCalReco_TICLSeedingRegion_h
+
+#include "DataFormats/HGCalReco/interface/Common.h"
+#include "DataFormats/Math/interface/normalizedPhi.h"
+#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
+#include "DataFormats/GeometryVector/interface/GlobalVector.h"
+#include "DataFormats/Provenance/interface/ProductID.h"
+
+namespace ticl {
```

Comment:
I think we have decided that if a class has TICL in the name, it does not also need to be in the ticl namespace.

Reply:
ok